scripts/_0_4_9_adept_experiment.py

```diff
@@ -0,0 +1,316 @@
+from decouple import config 
+import requests
+from scripts._0_4_4_adept_human_adm_compare_DRE_PH1_Server import main as dre_to_ph1
+from adept_repop_ph1 import SCENARIO_MAP
+import utils.db_utils as db_utils
+
+PH1_URL = config("ADEPT_URL")
+DRE_URL = config("ADEPT_DRE_URL")
+
+def main(db):
+    refill_PH1_sessions = 'http:' in PH1_URL # if ph1 server is local, refill sessions
+
+    # all DRE sessions should be in PH1 server already
+    if refill_PH1_sessions:
+        print("Repopulating Phase 1 server with DRE data")
+        dre_to_ph1(db)
+
+    update_surveys_with_opposite_server(4, db)
+
+    # PH1 sessions are not in DRE server!
+    # send all PH1 sessions to DRE server (my local DRE server was populated with no redacted files, and just one set of ph1 unredacted files)
+    print("Populating DRE server with Phase 1 data")
+    send_PH1_to_DRE(db)
+    update_surveys_with_opposite_server(5, db)
+
+
+def update_surveys_with_opposite_server(EVAL_NUMBER, db):
+    '''
+    Runs through all adept survey pages for EVAL_NUMBER and calculates
+    alignment using the opposite server (i.e. sending ph1 data to dre server, dre data to ph1 server)
+    '''
+    survey_collection = db['surveyResults']
+    plog = db['participantLog']
+    comparison_collection = db['humanToADMComparison']
+    url = DRE_URL if EVAL_NUMBER == 5 else PH1_URL
+
+    surveys = survey_collection.find({'results.evalNumber': EVAL_NUMBER})
+    for res in surveys:
+        # find valid surveys by valid participants
+        pid = res.get('results', {}).get('Participant ID Page', {}).get('questions', {}).get('Participant ID', {}).get('response')
+        if not pid:
+            continue
+        try:
+            in_plog = plog.count_documents({'ParticipantID': int(pid)})
+            if in_plog == 0:
+                continue
+        except:
+            continue
+
+        # go through each page to find the data we need
+        for pageName in res['results']:
+            if 'Medic' in pageName and ' vs ' not in pageName:
+                page = res['results'][pageName]
+                target = page['admTarget']
+                # only look for adept pages
+                if 'ADEPT' not in target:
+                    continue
+
+                # find comparison values from dre server from database
```

Comment:
Looks like this comment should be more generic as code is no longer DRE-specific.